docassemble/assemblylinewizard/data/questions/pdf_field_tester.yml

```diff
@@ -133,9 +125,14 @@ subquestion: |
   
   ${my_pdf}
   
-  **Note: you do not need to download this form.** This is just a preview
-  to help you catch any errors. Use the original PDF that you uploaded
-  for the next step.
+  [FILE stop_sign, 5%] **Do not download this form.** It is just a preview
+  to help you catch any errors. 
+  
+  [FILE stop_sign, 5%] **If your job is only to label the form, stop here**. Your form has passed the test, you can now pass the PDF file that you uploaded to the next person for interview building.
+  
+  [FILE green_light, 5%] **If your job is to build the interview**, press the "continue" button below to build your draft interview.
+continue button label: Continue to build interview
```

Comment:
There should be two buttons for the two "paths" of just labeling and building the whole interview. The continue button is good, but there should also be an exit button.

Reply:
Good idea, I'll add an exit button.